config/ses.toml

```diff
@@ -1,12 +1,24 @@
 [ses]
 
+# Whether or not to build rook from git. If not, the target version of SES images
+# are used.
+build_rook_from_git = false
+
+# Rook repo url
+repo = "https://github.com/SUSE/rook"
+
+# The rook version you want to test. This is the tag to be checked out so it must
+# exist for real within rook repo
+version = "master"
+
 # The target version of SES to install. This must match a key below that
 # contains the repositories and yaml substitutions.
 
 target = "devel7"
 
   [ses.ses7]
     rook_ceph_chart = "registry.suse.de/suse/sle-15-sp2/update/products/ses7/charts/rook-ceph:latest"
+    ceph_image = "registry.suse.de/devel/storage/7.0/containers/ses/7/rook/ceph:latest"
```

Comment:
This shouldn't be devel?

Reply:
Actually thinking about this more, why is the ceph_image part of this change? It seems unrelated to providing a custom rook_image. It could be a separate change though.

Reply:
When using self-build rook-images I do not want to use the rpm-package with the upstream or SES based yaml-files and helm charts. I want to use the ones out of my set rook-branch. Therefore I need to know which images for "fixing" the yaml-files.

Reply:
Right, I see now.

We could use the yaml_substitutions to achieve this, but that might be too complex? Maybe an example file would help if we did though.

If not, it might be worth noting that this is only used when building from git, and that the below yaml_substitutions are only used when /not/ building from git.